packages/catalog-search/src/SearchHeader.jsx

```diff
@@ -48,6 +49,7 @@ const SearchHeader = ({
             xs={12}
             md={8}
           >
+            { !hideSearchBox && (
```

Comment:
formatting nit: no whitespace after `{` and indent `SearchBox` to show it nested under the parentheses. 

```suggestion
            {!hideSearchBox && (
              <SearchBox
                className="mb-4"
                // ...
              />
            }
```